odoo/addons/test_main_flows/static/tests/tours/main_flow.js

```diff
@@ -990,7 +990,7 @@ tour.stepUtils.autoExpandMoreButtons('.o_control_panel .breadcrumb:contains("the
     position: 'bottom',
 }, {
     edition: "enterprise",
-    trigger: 'div[name=bank_statement_create_button] > a[data-name=create_bank_statement]',
+    trigger: 'div[name=bank_statement_create_button] > a[name=create_bank_statement]',
```

Comment:
add a version for legacy + the FIXME OWL Comment

Reply:
added a comment, and testing legacy and owl